### PR TITLE
[Web PubSub] Prepare packages for Nov release

### PR DESCRIPTION
**Pull request by Mohit-Chakraborty**

### CODE REVIEW

File: sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2021-11-09)
```
Comment (AlexGhiondea) [inline sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/CHANGELOG.md:3]: Should this be 1.0.0 ?

**Review by AlexGhiondea — APPROVED** [review]
Left a comment about a version, but LGTM otherwise!

File: sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/CHANGELOG.md (hunk 1 above)
Comment (Mohit-Chakraborty) [reply sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/CHANGELOG.md:3]: This one package is being released in beta. It is the first iteration of this package.

Comment (AlexGhiondea) [reply sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/CHANGELOG.md:3]: All right!

**Review by tg-msft — APPROVED** [review]

File: sdk/webpubsub/Azure.Messaging.WebPubSub/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,9 @@
...
+## 1.0.0 (2021-11-09)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Adds stable features and bug fixes from the preview releases.
```
Comment (vicancy) [inline sdk/webpubsub/Azure.Messaging.WebPubSub/CHANGELOG.md:6]: shall we list all the breaking changes here?

Comment (Mohit-Chakraborty) [reply sdk/webpubsub/Azure.Messaging.WebPubSub/CHANGELOG.md:6]: I thought about it. We can do that, but listing everything exhaustively would be a very long list, esp. with all the renames.

Comment (vicancy) [reply sdk/webpubsub/Azure.Messaging.WebPubSub/CHANGELOG.md]: Made some minor updates

**Pull request merged**